spk/python38/Makefile

```diff
@@ -0,0 +1,49 @@
+SPK_NAME = python38
+SPK_SHORT_VERS = 3.8
+SPK_VERS = $(SPK_SHORT_VERS).6
+SPK_REV = 1
+SPK_ICON = src/python3.png
+
+DEPENDS  = cross/busybox cross/$(SPK_NAME)
```

Comment:
Maybe that should be the right time to remove the busybox binary from this installation since it's kinda incompatible with DSM 7.0 securization stream, should it ?

Reply:
I agree. It was mainly delivered here to provide these generic service/tools to application packages (and reuse instead of rebuild)